bin/upload.js

```diff
@@ -38,16 +62,15 @@ actions = actions.filter((action) => {
 });
 
 // Validate that we have an API token.
-let token = process.env.npm_package_config_api_token;
-if (!token) {
-    console.error('No token specified, cannot upload actions without an API token.');
+if (!config.api_token) {
+    console.error(
+        `No token in ${config_env} config; cannot upload actions without an API token.`
```

Comment:
Maybe it would be cleaner in the future to have configuration checks like this all centralized into one place, when you load the config? On the other hand, having the assertions near the code that uses the code could make them easy to reason about.